### PR TITLE
React/stepper advanced

### PR DESCRIPTION
**Pull request by Hamza-Rafique**
Screen Shot 2021-08-05 at 5 27 20 PM

### CODE REVIEW

File: libs/react/src/lib/advanced-stepper/advanced-stepper.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+
+
+
+
```
Comment (fawadaslam777) [inline libs/react/src/lib/advanced-stepper/advanced-stepper.tsx:5]: Can you please remove extra spaces. We need to implement linting tools properly

File: libs/react/src/lib/advanced-stepper/advanced-stepper.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
```
Comment (fawadaslam777) [inline libs/react/src/lib/advanced-stepper/advanced-stepper.tsx:3]: I did not understand about these changes

File: libs/react/src/index.ts [hunk 3]
```diff
@@ -1 +1,3 @@
+
+export * from './lib/advanced-stepper/advanced-stepper';
```
Comment (fawadaslam777) [inline libs/react/src/index.ts:2]: file should start from line one. what do you think?

**Review by fawadaslam777 — COMMENTED** [review]
only need to remove the extra lines and comments

**Review by fawadaslam777 — APPROVED** [review]

**Review by zeenainali — APPROVED** [review]

**Pull request merged**